webapp/components/Comment/Comment.vue

```diff
@@ -79,6 +86,7 @@ export default {
       isTarget,
       isCollapsed: !isTarget,
       openEditCommentMenu: false,
+      answered: this.$t('post.comment.answered'),
```

Comment:
Okay, now I understand.
Please use `this.$t('post.comment.answered‘)` directly in `:title="this.$t('post.comment.answered')“` otherwise it seems not to be responsive if you declare it as `data`.

Please rename the variable to `answer`, because it is what you do right now and not what you have done already …

Reply:
ok

Reply:
@ogerly I think `reply` would fit here.

Reply:
ok